FinSearchUnified/Tests/BusinessLogic/Models/FindologicArticleModelTest.php

```diff
@@ -200,9 +195,15 @@ public function testArticleKeywords()
         $keywords = [];
 
         foreach ($properties->getValue($xmlArticle)->getValues() as $value) {
-            $keywords = array_merge($keywords, array_map(function ($item) {
-                return $item->getValue();
-            }, $value));
+            $keywords = array_merge(
```

Comment:
This is actually unreadable, please revert this to how it was before.

Reply:
Actually this is the PSR12 code style.